orbs.rb

```diff
@@ -0,0 +1,23 @@
+require "sinatra"
+require 'sinatra/json'
+require "sinatra/reloader" if development?
+require "sequel"
+require_relative "db.rb"
+
+
+DB = Sequel.connect(adapter: 'mysql2', host: HOST, database: DBNAME, user: USERNAME, password: PASSWORD)
+get '/' do
+    DB.fetch("SELECT name,inet_ntoa(ip) as ip,water_uuid, elec_uuid, elec_rvid, water_rvid, relative_value FROM orbs o inner join relative_values rv on o.water_rvid=rv.id").map do |row|
+      content_type :json
+      output = {
+        :name=>row[:name],
+        :ip=>row[:ip],
+        :water_uuid=>row[:water_uuid],
+        :elec_uuid=>row[:elec_uuid],
+        :elec_rvid=>row[:elec_rvid],
+        :water_rvid=>row[:water_rvid],
+        :relative_value=>row[:relative_value]
+      }
+      output.to_json
```

Comment:
Considering `output` is only used once here, why declare the variable at all?

Reply:
I don't really understand what the alternative would be.

Reply:
The last line in any Ruby method is what is returned.  Do we really need to save `output` into a variable, or can we condense this into one line that may make it easier to read / more efficient?